modules/dataverse-spi/src/main/java/io/gdcc/spi/export/XMLExporter.java

```diff
@@ -0,0 +1,43 @@
+package io.gdcc.spi.export;
+
+import javax.ws.rs.core.MediaType;
+
+/**
+ * XML Exporter is an extension of the base Exporter interface that adds the
+ * additional methods needed for generating XML metadata export formats.
+ */
+public interface XMLExporter extends Exporter {
+
+    /**
+     * @return - should always be true for XML Exporters
+     */
+    default Boolean isXMLFormat() {
+        return true;
+    };
+
+    /**
+     * @implNote for the ddi exporter, this method returns "ddi:codebook:2_5"
+     * @return - the name space of the XML schema
+     */
+    String getXMLNameSpace();
+
+    /**
+     * @implNote for the ddi exporter, this method returns
+     *           "https://ddialliance.org/Specification/DDI-Codebook/2.5/XMLSchema/codebook.xsd"
+     * @return - the location of the XML schema
+     */
+    String getXMLSchemaLocation();
+
+    /**
+     * @implNote for the ddi exporter, this method returns "2.5"
```

Comment:
Isn't that an URI according to the XML Spec?

Reply:
Yes. In the code (not changed in this PR), it is basically cut/pasted as a String. I've added a comment in the interface Javadoc to note that it has to be a valid URI. (The code could be changed to enforce a URI, but I'd ask that that be done as a separate PR if its a priority.)

Reply:
How often and how frequent can we change the API and expect external develops to adapt their code? I would prefer to integrate this into this MR so there is a well-defined API that won't change in the near future...